### PR TITLE
micro:bit release notes 0.12.22

### PR DESCRIPTION
**Pull request by pelikhan**
- [x] need gif for slow-mo
- [ ] getting started UI changes
- [x] Language picker

### CODE REVIEW

**Comment by abchatra** [comment]
Some of the things missing: 
- getting started UI changes
- Language picker

**Comment by pelikhan** [comment]
Passing on the token.

**Comment by pelikhan** [comment]
looking at the language stuff

File: docs/blog/microbit/v0.12.22.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+## Grid pickers
+
+The gesture and pin dropdown have been replaced with grid pickers to avoid having to scroll down to find the
+correct option!
+
+![](/static/blog/microbit/v0.12.22/grid.gif)
+
+## API updates
+
+* `input.calibrateCompass()` is now available to explicitely start the calibration process for the compass.
```
Comment (riknoll) [inline docs/blog/microbit/v0.12.22.md]: explicitely => explicitly

File: docs/blog/microbit/v0.12.22.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+* sprite deletion issue fixed
+* `serial.readBuffer` and `serial.writeBuffer` allow to read/write binary data over serial
+* `bluetooth.onUartDataReceive` event raised when UART data is coming in via BLE
+* support for specifying time interval in `basic.showIcon` and `basic.showArrow` similarly to `basic.showLeds`
```
Comment (riknoll) [inline docs/blog/microbit/v0.12.22.md]: **the** time interval

similarly => similar

File: docs/blog/microbit/v0.12.22.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
+# Micro:bit Editor v0.12.22
+
+This release introduces **Arrays** in blocks, **Slow-Mo** in simulator, better **Sidedocs**, as well as various bug fixes and improvements.
```
Comment (riknoll) [inline docs/blog/microbit/v0.12.22.md]: **the** simulator

**Comment by microsoftsam** [comment]
Is there any reason why this is going to v0? The main site is hosted off master.

**Comment by pelikhan** [comment]
Just working off v0 while on microbit land. Will cherry pick into master afterwards.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by microsoftsam** [comment]
Okay cool.

**Comment by pelikhan** [comment]
@DavidWhaleMEF  please review the changes.

**Pull request merged**